README.md

```diff
@@ -78,7 +78,8 @@ or the configuration file can be specified by cli args as well.
 * `make run-mock-registry`: Mock registry. Entirely separate binary.
 * `make test`: Runs the test suite.
 * `make vendor`: Updates the dependencies
-* `make build`: Builds a docker container of the current source
+* `make dev-image`: Builds a docker container of the current source
+* `make release-image` Builds a docker container using the latest rpm from [Copr](https://copr.fedorainfracloud.org/coprs/g/ansible-service-broker/ansible-service-broker/)
```

Comment:
@jmrodri Ya the reason I think of docker build is because the main way to run this stuff is in the container. Where ```make build``` translates to: get my code ready for me to run it. Those steps are compile the code and build and image.